infra/modules/aws/variables.tf

```diff
@@ -48,7 +48,7 @@ variable "caller_aws_arns" {
 variable "psoxy_version" {
   type        = string
   description = "version of psoxy to deploy"
-  default     = "0.4.4"
+  default     = "0.4.5-jrc"
```

Comment:
the `-jrc` thing confuses me

Reply:
to clarify: was a "jose release candidate" to not collide with any other, did that temporarily to check validity of terraform files.